tests/integration/test_apply_checks.py

```diff
@@ -4299,6 +4300,7 @@ def test_apply_checks_all_row_checks_as_yaml_with_streaming(ws, make_schema, mak
                 datetime(2025, 1, 12, 2, 0, 0),
                 {"key1": 1},
                 {"field1": 1},
+                "0.0.0.0",
```

Comment:
This makes `test_apply_checks_all_row_checks_as_yaml_with_streaming` to fail. Update it so that it falls within the CIDR: '192.168.1.0/24', e.g. "192.168.1.2",